mica-rest/src/main/java/org/obiba/mica/study/rest/DraftHarmonizationStudyResource.java

```diff
@@ -206,6 +208,30 @@ public FileResource file(@PathParam("fileId") String fileId, @QueryParam("key")
     return dtos.asDto(studyService.getFromCommit(studyService.findDraft(id), commitId), true);
   }
 
+  @GET
+  @Path("/diff")
+  public Response diff(@NotNull @QueryParam("left") String left, @NotNull @QueryParam("right") String right) {
```

Comment:
Can't we refactor this method instead of repeating it in each service, may be in the base class?